packages/containers/src/SubHeaderBar/SubHeaderBar.container.js

```diff
@@ -50,20 +50,32 @@ class SubHeaderBar extends React.Component {
 	render() {
 		const state = this.props.state || DEFAULT_STATE;
 		const hasGoBack = this.props.onGoBack || this.props.actionCreatorGoBack;
+		const { onEdit, onCancel, onSubmit, onChange } = this.props;
+		const eventHandlerProps = {};
+		if (onEdit) {
+			eventHandlerProps.onEdit = onEdit;
+		}
+		if (onCancel) {
+			eventHandlerProps.onCancel = onCancel;
+		}
+		if (onSubmit) {
+			eventHandlerProps.onSubmit = onSubmit;
+		}
+		if (onChange) {
+			eventHandlerProps.onChange = onChange;
+		}
+		if (hasGoBack) {
+			eventHandlerProps.onGoBack = this.onGoBack;
+		}
 		const props = Object.assign(
 			{},
 			omit(this.props, cmfConnect.INJECTED_PROPS),
-			{
-				onEdit: this.props.onEdit,
-				onCancel: this.props.onCancel,
-				onSubmit: this.props.onSubmit,
-				onChange: this.props.onChange,
-				onGoBack: hasGoBack && this.onGoBack,
-			},
+			eventHandlerProps,
```

Comment:
I'm not sure why changes in this file were made. Personally I found the old one easier to read than the `if` blocks. I'm not sure of the actual difference of not passing undefined values here so maybe there's something I'm missing.

Reply:
it's all about it, you are right, 
avoiding passing `undefined` props that override the default behavior